en/docs/install-and-setup/setup/api-controller/managing-rate-limiting-policies/migrating-rate-limiting-policies-to-different-environments.md

```diff
@@ -0,0 +1,57 @@
+# Importing a rate limiting policy from one environment to another using APICTL
```

Comment:
Let's be consistent with this doc [1]. Topics and all

[1] https://apim.docs.wso2.com/en/4.2.0/install-and-setup/setup/api-controller/managing-common-api-policies/migrating-common-api-policies-to-different-environments/

Reply:
Fixed.